src/shared/database/index.ts

```diff
@@ -72,6 +72,7 @@ async function setupCrawlsTable(): Promise<void> {
       table.string('ip')
       table.integer('port')
       table.string('ws_url')
+      table.boolean('connected')
```

Comment:
This requires an alteration to the database. We'll want to be certain that we record the process for deploying to stage, and be careful deploying to prod.

Reply:
https://www.postgresql.org/docs/12/sql-altertable.html